shcDecoder/src/main/java/ca/bc/gov/shcdecoder/SHCVerifierImpl.kt

```diff
@@ -94,6 +95,8 @@ class SHCVerifierImpl(
         var mrnType = 0
         var nrvvType = 0
         var winacType = 0
+        var minInterval = 0
```

Comment:
What's this variable for? I don't see it being used

Reply:
minInterval = ruleSet?.minDays ?: 0